src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -168,29 +138,18 @@ public PluginCompatReport testPlugins() throws IOException {
                 && localCheckoutProvided()
                 && !data.plugins.containsKey(config.getIncludePlugins().get(0))) {
             String artifactId = config.getIncludePlugins().get(0);
-            try {
-                UpdateSite.Plugin extracted = extractFromLocalCheckout();
-                data.plugins.put(artifactId, extracted);
-            } catch (PluginSourcesUnavailableException e) {
-                LOGGER.log(
-                        Level.SEVERE,
-                        "Cannot test {0} because plugin sources are not available despite a local"
-                                + " checkout being provided",
-                        artifactId);
-            }
```

Comment:
"Log and drive on" is an antipattern. Immediately throw this up to implement the philosophy of "we don't swallow exceptions around here."